tests/integ/conftest.py

```diff
@@ -63,26 +79,71 @@ def nonce(request):
     return request.config.getoption("--nonce")
 
 
-@pytest.yield_fixture(autouse=True)
-def cleanup_objects(engine):
```

Comment:
To be clear, this is gone because dynamodb local stores everything in memory?

Reply:
Correct

Reply:
Going to add this back in because this doesn't run dynamodb-local for every test, and that would be a huge waste of time.